src/Bridge/CommonMark/CommonMarkParser.php

```diff
@@ -3,23 +3,23 @@
 namespace Mni\FrontYAML\Bridge\CommonMark;
 
 use League\CommonMark\CommonMarkConverter;
-use League\CommonMark\MarkdownConverterInterface;
+use League\CommonMark\ConverterInterface;
 use Mni\FrontYAML\Markdown\MarkdownParser;
 
 /**
  * Bridge to the League CommonMark parser
  */
 class CommonMarkParser implements MarkdownParser
 {
-    private MarkdownConverterInterface $parser;
+    private ConverterInterface|null $parser;
```

Comment:
Same here, this is never null

Reply:
This has now been resolved in 7f27a4c25ee0b174b5790af6dec236c9c4c5a38a.